cmd/cstor-pool-mgmt/controller/common/common.go

```diff
@@ -57,6 +57,10 @@ const (
 	MessageResourceFailCreate EventReason = "Resource creation failed"
 	// SuccessImported holds status for corresponding imported resource.
 	SuccessImported EventReason = "Imported"
+	// FailureImported holds status for corresponding imported resource.
+	FailureImported EventReason = "Import failure"
+	// FailureImportOperations holds status for corresponding imported resource.
+	FailureImportOperations EventReason = "Failure Import operations"
```

Comment:
Can this be `Import operations failure`?

In addition, is this the right time to change the variable names?
e.g. `ReasonImportOperationsFailure`, `ReasonImportFailure`, `ReasonImportSuccessful`, `ReasonCreateResourceFailure`, `ReasonImportResourceSuccessful`

Naming convention that i have assumed here is as follows:
`Reason<Action><Entity/Adjective of the Entity><Sucess/Failure>`

Reply:
Are these reasons used across by other controllers ?
In that case, we need to ensure all controllers make use of same constants.